.github/workflows/continuous_integration.yml

```diff
@@ -0,0 +1,42 @@
+name: Continuous Integration
+on:
+  push:
+    branches:
+    - master
+    paths:
+    - 'actions/**'
+    - 'Dockerfile'
+
+jobs:
+  docker:
+    name: Build Action Server Docker Image
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout git repository 🕝
+        uses: actions/checkout@v2
+
+      - name: Authenticate into Google Cloud Platform
+        uses: GoogleCloudPlatform/github-actions/setup-gcloud@master
+        with:
+          version: '275.0.0'
+          service_account_key: ${{ secrets.GCLOUD_AUTH }}
+
+      - name: Configure Docker to use Google Cloud Platform
+        run: |
+          gcloud auth configure-docker
+
+      - name: Pull Latest Image
+        run: |
+          docker pull gcr.io/replicated-test/rasa-demo:latest || true
+
+      - name: Set Build ID from run ID and number
+        run: echo ::set-env name=BUILD_NUMBER::$GITHUB_RUN_NUMBER-$GITHUB_RUN_ID
+
+      - name: Build Image
+        run: |
+          docker build -t gcr.io/replicated-test/rasa-demo:run$BUILD_NUMBER -t gcr.io/replicated-test/rasa-demo:latest --cache-from gcr.io/replicated-test/rasa-demo:latest .
+      
+      - name: Push PR Image to Google Cloud Container Registry
+        run: |
+          docker push gcr.io/replicated-test/rasa-demo:run$BUILD_NUMBER
```

Comment:
How does it also push to latest? Looks like we tag as both but only push to the build number. 

https://console.cloud.google.com/gcr/images/replicated-test/GLOBAL/rasa-demo?project=replicated-test&gcrImageListsize=30 doesn't really list latest

Reply:
yeah I guess it doesn't... I thought by tagging it as both, when it was pushed as one it would also be tagged as the other. Should  we do both?

Reply:
the last `latest` is from jan so no... it's not tagging it as that

Reply:
added latest as well

Reply:
actually! pushing just `rasa-demo` should push both tags. Tested it on helpdesk https://hub.docker.com/r/rasa/helpdesk-assistant/tags

Reply:
this is really cool, do you think it makes it harder to figure out which tag corresponds to which pr though?